openmetadata-docs-v1/content/v1.0.0/deployment/kubernetes/gke.md

```diff
@@ -112,7 +112,7 @@ kubectl create -f nfs-cluster-ip-service.yml
 
 We create a CluserIP Service for pods to access NFS within the cluster at a fixed IP/DNS.
 
-</Collapse>
+{%/collapse%}
```

Comment:
This tag is not [here](https://github.com/open-metadata/docs-v1/tree/publish/markdoc/tags) , so I don't think it will work.

Reply:
Oh yes. Thanks 👍